server/lib/configuration.js

```diff
@@ -147,6 +147,20 @@ const conf = module.exports = convict({
       'development'
     ]
   },
+  featureFlags: {
+    implementation: {
+      default: 'launch-darkly',
+      doc: 'The feature-flagging implementation to use, e.g. "launch-darkly" or "unleash"',
+      env: 'FEATURE_FLAG_IMPLEMENTATION',
+      format: String
```

Comment:
would it make sense to make this a `format` like https://github.com/mozilla/fxa-content-server/blob/b8e01a4ae15a82a310f3a7f4a2a64e27d321cf4b/server/lib/configuration.js#L395 and pick from a list of available options? Also would it make sense to introduce a `none` to turn all flags off, is there a scenario where we would need this?